src/models/facility.model.ts

```diff
@@ -29,10 +29,10 @@ const FacilitySchema: Schema = new Schema({
   Operations: { type: String, required: true },
   Reviews: [{ type: mongoose.Schema.Types.ObjectId, ref: "Review" }],
   Date: { type: Date, required: true },
-  PictureURL: { type: String, required: false },
+  PictureURL: { type: [String], required: false },
   RatingAVG: { type: Number, required: true },
   Favorites: { type: Number, required: true },
```

Comment:
Shouldn't Favorites be false as well especially as when a facility is first created it wouldn't be favorited by any?